velox/row/CompactRow.h

```diff
@@ -108,6 +118,12 @@ class CompactRow {
   /// Serializes struct value to buffer. Value must not be null.
   int32_t serializeRow(vector_size_t index, char* buffer);
 
+  /// Serializes struct value to buffer. Value must not be null.
+  void serializeRow(
```

Comment:
serializeRow --> serializeRows

Reply:
The naming of `serializeRow` is to serialize the `RowVector`. For map and array, there are `serializeArray` and `serializeMap`